symfony/composer.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "algolia/api-clients-playground",
+    "type": "project",
+    "description": "The Symfony playground.",
+    "keywords": [
+        "framework",
+        "symfony"
+    ],
+    "license": "MIT",
+    "require": {
+        "php": "^7.1.3",
+        "algolia/search-bundle": "^3.4",
+        "symfony/console": "4.2.*",
```

Comment:
This is wrong, you should require ^3.2 instead. https://semver.org. Same goes for the rest of them.

Don't forget that laravel does'nt follow semver.